#6 - Configure quota with application properties

AtoDaX opened the pull request:
I couldn't make map immutable without using class where this map would be a field

zhukovsd reviewed (changes requested):
@AtoDaX 

- [x] Immutable/unmodifiable bean with quotas

AtoDaX committed:
feat: created a Map<String, Double> bean with quotas from configuration files.
Bean added in LimiterController for test purposes.
Get request on /quota/{operation_id} where operation_id is key prints value to console.

AtoDaX committed:
feat: rpsQuotas bean now unmodifiable

zhukovsd committed:
gitignore target directory

zhukovsd committed:
QuotasConfiguration - suppress `ConfigurationProperties` warning

zhukovsd reviewed (approved):
All good.
